content/webapp/components/GuideStopCard/index.tsx

```diff
@@ -45,26 +45,12 @@ const GuideStopCard: FunctionComponent<Props> = ({
     ? secondsToHoursMinutesAndSeconds(durationInSeconds)
     : undefined;
   const croppedImage = getCrop(image, '16:9');
-  const placeholderBackgroundColor = (stopNumber: number): ColorSelection => {
-    switch (stopNumber % 4) {
-      case 0:
-        return 'accent.salmon';
-      case 1:
-        return 'accent.blue';
-      case 2:
-        return 'accent.purple';
-      case 3:
-      default:
-        return 'accent.green';
-    }
-  };
-
   return (
     <Space
       $v={{ size: 'l', properties: ['margin-bottom'] }}
       className={grid(threeUpGridSizesMap.default[0])}
     >
-      <CardOuter href={link} style={{ minHeight: '0' }}>
+      <CardOuter href={link} style={{ minHeight: '0' }} id={`${number}`}>
```

Comment:
adding `id` so we can link back to the right point in the list of stops when you 'close' the individual stop

Reply:
Nice touch

Reply:
Makes sense! 👍 
NP: could just be `id={number}`, mostly caught my eye because I thought the [added brackets rule](https://github.com/wellcomecollection/wellcomecollection.org/pull/11028/files) would've flagged it? Very much a non-blocker.

Reply:
good point, will update

Reply:
…oh wait, I remember now – this is TS BS. It complains that an html attribute has to be a string, and `number` is a… number. So I wrapped it in quotes.

Reply:
Ohhhh that makes sense.